data/campaigns/Eastern_Invasion/scenarios/10_Dark_Sanctuary.cfg

```diff
@@ -0,0 +1,2776 @@
+                            [/modify_side]
+
+                            {REPLACE_SCENARIO_MUSIC vengeful.ogg}
+
+                            {CLEAR_VARIABLE sanctum_antechamber_open}
+                            {VARIABLE inside_inner_sanctum yes}
+                            [show_objectives]
+                            [/show_objectives]
+                        [/command]
+                    [/option]
+                    [option]
+                        label= _ "No, not yet."
+                        [command]
+                            # do nothing
+                        [/command]
+                    [/option]
+                [/message]
+            [/then]
+        [/if]
+    [/event]
+
+    #---------------------------
+    # ORCISH ATTACKS
+    #---------------------------
+    [event]
+        name=turn 9
+        {REPLACE_SCENARIO_MUSIC northerners.ogg}
+
+        {NAMED_UNIT 2 (Orcish Sovereign) 50 30 _"Chief Dra-Nak" (_"Chief Dra-Nak") ()}
+        [+unit]
+            profile=portraits/orcs/ruler-2.webp
+            canrecruit=yes
+            [modifications]
+                {TRAIT_INTELLIGENT}
+                {TRAIT_STRONG}
+            [/modifications]
+        [/unit]
+        {SCROLL_TO 50 30}
+
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "What do we have here? This is what those wyrms were groveling about?"
+        [/message]
+        [message]
+            speaker="Chief Dra-Nak"
+            #po: he's a chief from the Whitefangs because he's not the only one, just a particularly powerful one
+            message= _ "I’m Dra-Nak, Chief from the Whitefang clan. What’re you doing so far away from home, human?"
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "I am a lord of the Horse Clans, and we are here on Wesnoth business. You would do well not to bother us."
+        [/message]
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "Wesnoth business, huh. ...you’re not the first group of deserters I’ve caught fleeing the undead."
+        [/message]
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "I’ll tell you what. I’m a fair orc, I won’t advantage of your situation. Maybe I can help you. Let’s make a deal."
+        [/message]
+        [message]
+            speaker="Chief Dra-Nak"
+            message= _ "Those necromancers down south are paying me good coin to keep them supplied with corpses. Hand over, let’s say, eight of your soldiers as tribute, and you’re free to go about your business. I’ll even give you a cut of the profit!"
+        [/message]
+        [message]
+            speaker=Owaec
+            message= _ "Today I have been called shortsighted, then a coward, and now a deserter!?! My honor will stand no more insult! Begone, fetid orc, lest I mount your head on a pike! I don’t fear the dead and I don’t fear you."
```

Comment:
No need for the second exclamation mark after 'deserter'. Although for consistency maybe it should be '?!' as it is in other locations.